docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>Using UXarray we can construct the dual mesh using <code>grid.compute_dual()</code> , which returns a new grid object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>You want me to replace everything with just this?</p>

Reply:
<p>Whoops, meant only the final portion. </p>